### PR TITLE
Add a note about compositional fields to the manual.

### PR DESCRIPTION
**Pull request by bangerth**

### CODE REVIEW

File: doc/manual/manual.tex [hunk 1]
```diff
@@ -1555,6 +1555,19 @@ \subsection{Compositional fields}
 all sorts of components of models that go beyond the simple Stokes plus
 temperature set of equations. Play with them!
 
+\note{As has hopefully become clear from the discussion above, the
+  term ``compositional field'' as used in \aspect{} is by now mostly
+  historic: These fiels were meant to track chemical compositions, but
```
Comment (bobmyhill) [inline doc/manual/manual.tex:1560]:
```suggestion
  historic: These fields were meant to track chemical compositions, but
```

File: doc/manual/manual.tex [hunk 2]
```diff
@@ -1555,6 +1555,19 @@ \subsection{Compositional fields}
...
+  are now used to track all sorts of other things as well, or in some
+  cases track nothing at all and just be static fields that do not
+  change over time and whose values may simply indicate where some
+  features of the model are located.
+
+  It is therefore useful to think of the term ``compositional field''
+  as simply a \textit{technical term} in which the two words appear
```
Comment (bobmyhill) [inline doc/manual/manual.tex:1567]:
```suggestion
  as a \textit{technical term} in which the two words appear
```

File: doc/manual/manual.tex [hunk 3]
```diff
@@ -1555,6 +1555,19 @@ \subsection{Compositional fields}
 all sorts of components of models that go beyond the simple Stokes plus
 temperature set of equations. Play with them!
 
+\note{As has hopefully become clear from the discussion above, the
+  term ``compositional field'' as used in \aspect{} is by now mostly
+  historic: These fiels were meant to track chemical compositions, but
+  are now used to track all sorts of other things as well, or in some
+  cases track nothing at all and just be static fields that do not
```
Comment (bobmyhill) [inline doc/manual/manual.tex:1562]:
```suggestion
  cases track nothing at all and just be static fields that
```

File: doc/manual/manual.tex [hunk 4]
```diff
@@ -1555,6 +1555,19 @@ \subsection{Compositional fields}
...
+  change over time and whose values may simply indicate where some
```
Comment (bobmyhill) [inline doc/manual/manual.tex:1563]:
```suggestion
simply indicate where some
```
Remove tautology

**Review by bobmyhill — CHANGES_REQUESTED** [review]

**Comment by bangerth** [comment]
Thanks, @bobmyhill -- so fixed!

**Review by bobmyhill — APPROVED** [review]

**Pull request merged**